docs/Other/marketplace_webhooks.md

```diff
@@ -774,6 +774,14 @@ title: Customer Delete Event
 
 The spend change request webhook is called every time a spend change is requested for one of the vendor's products.
 
+Details:
+
+- **currency** _The currency in which the spend/budget is being requested._
+- **requested_value** _The requested new spend value/budget, in cents (or smallest denomination in other currencies)._

```

Comment:
Without seeing this in context it's difficult for me to determine what might be the "most" clear phrasing haha. But off the top of my head:

Received from reseller
- currency: The currency of the requested spend
- conversion_value: The amount of the requested spend in cents (or smallest denomination in other currencies), before currency conversion

Sent to vendor
- conversion_currency: The currency that the requested spend will be converted to
- requested_value: The amount of the requested spend in cents (or smallest denomination in other currencies) after conversion to the conversion_currency

This might be kind of off base though, please let me know if I misinterpreted the request for input

Reply:
That sounds great, thanks!